Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Execution_of_an_uncommon_process_at_an_early_startup_stage.yml

```diff
@@ -0,0 +1,1783 @@
+      script: '|||file'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      file:
+        complex:
+          root: alert
+          accessor: targetprocesssha256
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: alert.initiatorsha256
+                iscontext: true
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: alert.cgosha256
+                iscontext: true
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -4
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "42":
+    id: "42"
+    taskid: f0584606-75ed-4d03-849a-c1af5d928a17
+    type: title
+    task:
+      id: f0584606-75ed-4d03-849a-c1af5d928a17
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1432,
+          "y": 2007
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "43":
+    id: "43"
+    taskid: 357da12c-5fcc-4eec-8dbf-ff8f712069e4
+    type: title
+    task:
+      id: 357da12c-5fcc-4eec-8dbf-ff8f712069e4
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "53"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -260
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "46":
+    id: "46"
+    taskid: dde788b4-17f7-42dc-b3f7-48a8ff160944
+    type: condition
+    task:
+      id: dde788b4-17f7-42dc-b3f7-48a8ff160944
+      version: -1
+      name: Does the command line have a medium or higher risk level?
+      description: This task evaluates the command-line analysis results and checks if the risk score is greater than or equal to 30. If it is, mark the result as **Malicious**.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "50"
+      Malicious:
+      - "3"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: greaterThanOrEqual
```

Comment:
Did you check if the condition works when there are multiple values returned?
Alert 12237 -  seems like an old version of the playbook.

Please verify that both Malicious and condition are answered.

Reply:
Done. It's working properly.